src/Project/JobCommand.ts

```diff
@@ -37,6 +38,7 @@ class JobCommand implements Job {
     this.jobType = Job.Type.tUndefined;
     this.name = '';
     this.valid = false;
+    this.workDir = require('os').homedir();
```

Comment:
I felt assigning as homedir is better to implemented in each child classes' constructor.

+) I'm not familiar about the history of `JobCommand`. Does `JobCommand` is designed to be a parent class of *workspace independent* commands only, such as `install`, `uninstall`, etc.? If it is, never mind the suggestion above.

Reply:
> Does `JobCommand` is designed to be a parent class of *workspace independent* commands only, such as `install`, `uninstall`, etc.? 

The original design of `JobCommand` depended on workspace. Because it was only used to run compiler. There was no workflow about Toolchain at the initial version. However, the design has been changed. We use `JobCommand` to request job related to Toolchain and to run compile. If it requests to run Toolchain job, it can run workspace independent. And if it requests to run Compile job, it should run at the `cfg` location. So I defined the `workDir` of `JobCommand` as os home directory and only `JobConfig` job has different workDir as per its cfg location.

I will apply your suggestion.

Reply:
AFAIU `JobCommand` is designed for toolchain workflow, and it can run workspace independent. 

IMO it's okay for current status. :smile: Now, the child classes should define `workDir` if the child process needs specific working directory. Let's go which this.

Reply:
After talking with @yunjayh, we agreed to keep the code as it is.